lib/client/db/dbcmd/dbcmd.go

```diff
@@ -505,6 +510,26 @@ func (c *CLICommandBuilder) getSnowflakeCommand() *exec.Cmd {
 	return cmd
 }
 
+func (c *CLICommandBuilder) getElasticsearchCommand() *exec.Cmd {
```

Comment:
```suggestion
func (c *CLICommandBuilder) getElasticSearchCommand() *exec.Cmd {
```

Reply:
Actually `Elasticsearch` seems to be the preferred capitalization: see Elastic docs, e.g. https://www.elastic.co/guide/en/elasticsearch/reference/master/index.html.

Reply:
my bad, Thought that ElasticSearch is commonly uses but after a second check the `Elasticsearch` is correct.